openwisp_controller/config/base/whois.py

```diff
@@ -51,7 +51,11 @@ class Meta:
     def clean(self):
         if ip_address(self.ip_address).is_private:
             raise ValidationError(
-                _("WhoIs information cannot be created for private IP addresses.")
+                {
+                    "ip_address": _(
+                        "WhoIs information cannot be created for private IP addresses."
```

Comment:
We said to use "WHOIS" right? Let's stick to it. I recommend using case sensitive search and replace in your editor to spot all of these.